### PR TITLE
Devx 856 link purchase finance

### PR DESCRIPTION
**Pull request by Astro-Choco**
- Added a user guide about linking between purchases and transactions
- Updated overview.md with links
- Updated fetch-account-transactions.md with examples

### CODE REVIEW

File: finance-api/overview.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
 Finance API overview
-======
-Merchants who take payments with Zettle have a Zettle account. The account processes transactions for the payments. For example, card payments made with Zettle card readers.
+===
+Merchants who take payments with Zettle have an Zettle account that processes transactions for the payments. For example, card payments made with Zettle card readers.
 
 The Finance API is used to fetch finance-related information that is processed through Zettle. The information includes account balance, all the transactions processed through Zettle, and payout.
 
-## Understand how payments work at Zettle
+# Understand how payments work at Zettle
```
Comment (AndreaFilyo) [inline finance-api/overview.md:7]:
```suggestion
# Understand how payments with Zettle work
```

File: finance-api/user-guides/fetch-account-transactions.md [hunk 2]
```diff
@@ -138,10 +138,10 @@ Send a request where you specify transaction types as you need. See [supported t
    ```
    GET /organizations/self/accounts/liquid/transactions?start={start_date}&end={end_date}&includeTransactionType={includeTransactionType}
    ```
-
-   Example:
+### Fetch card payment fee
+The following example fetches all card payments and the associated card payment fees from the merchant's liquid account from 1 January, 2020 to 31 December, 2020.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-account-transactions.md:142]:
```suggestion
The following example fetches all card payments and associated card payment fees from the merchant's liquid account. The transactions are fetched from 1 January, 2020 to 31 December, 2020.
```

File: finance-api/user-guides/fetch-account-transactions.md [hunk 3]
```diff
@@ -170,9 +170,42 @@ Send a request where you specify transaction types as you need. See [supported t
     }
    ```
 
+### Fetch transactions for payout
+The following example fetches all the transactions that are are included in the payout from the merchant's liquid account from 6 September, 2020 to 10 September, 2020.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-account-transactions.md:174]:
```suggestion
The following example fetches all transactions included in the payout from the merchant's liquid account. The transactions are fetched from 6 September, 2020 to 10 September, 2020.
```

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
+Fetch purchase information for transactions
+===
+Using the Finance API and the Purchase API, you can fetch purchase information for transactions. The `originatingTransactionUuid` of a transaction in the Finance API corresponds to the `uuid` of `payments` in the Purchase API.  
+> **Note:** Currently, fetching purchase information is available only for transactions that are made with cards and alternative payment methods (APMs) like PayPal QRC through Zettle.
+
+The Finance API records transactions for payment types that are supported by the Purchase API except `IZETTLE_CASH`, `SWISH`, `VIPPS`, `MOBILE_PAY`, `GIFTCARD`, `STORE_CREDIT`, and `IZETTLE_INVOICE`. 
+> **Note:**  If `IZETTLE_INVOICE` is paid with a card, the purchase will be recorded as `CARD_PAYMENT` in the Finance API.
+
+The following table shows the payment methods, payment types that are supported by the Purchase API, and the corresponding transaction types that are supported by the Finance API.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-purchase-information-for-transactions.md:9]:
```suggestion
The following table shows the payment methods and types supported by the Purchase API, and the corresponding transaction types supported by the Finance API. 
```

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
+Fetch purchase information for transactions
+===
+Using the Finance API and the Purchase API, you can fetch purchase information for transactions. The `originatingTransactionUuid` of a transaction in the Finance API corresponds to the `uuid` of `payments` in the Purchase API.  
+> **Note:** Currently, fetching purchase information is available only for transactions that are made with cards and alternative payment methods (APMs) like PayPal QRC through Zettle.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-purchase-information-for-transactions.md:4]:
```suggestion
> **Note:** Currently, fetching purchase information is available only for transactions made with cards and alternative payment methods (APMs) like PayPal QRC through Zettle.
```

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorisation is set up with the following OAuth scope using [OAuth2 API](../../authorization.adoc):
+    * `READ:FINANCE`
+    * `READ:PURCHASE`
+
+## Step 1: Find the transaction UUID with the Finance API
+Currently, fetching purchase information is available only for transactions that are made with cards, PayPal QRC, and Klarna.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-purchase-information-for-transactions.md:30]:
```suggestion
Currently, fetching purchase information is available only for transactions made with cards, PayPal QRC, and Klarna. 
```

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+                "originatingTransactionUuid": "e548933c-53fb-4a99-bbc5-c31f7861bcc3"
+            }
+        ]
+    }
+    ```
+
+3. In the response, find and save the value of `originatingTransactionUuid` for transactions for which you want to fetch purchase information.
+
+## Step 2: Fetch purchase information for transactions with the Purchase API
+Using the value of `originatingTransactionUuid` of a transaction in the Finance API, you can use the Purchase API to locate the purchase information for the transaction.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-purchase-information-for-transactions.md:84]:
```suggestion
With the `originatingTransactionUuid` value from a Finance API transaction, you can use the Purchase API to locate the purchase information for the transaction.
```

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+
+> **Note:** For refunds, a new "refund purchase" record is created with a negative amount instead. It represents a refund of an original purchase. Transactions of the type `CARD_REFUND` in the Finance API refers to this new "refund purchase" instead of the original purchase.
+
+1. Regularly fetch purchases and store them in a local database. For example, every midnight. In the request parameters, set the `startDate` to the last time of fetching purchases.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-purchase-information-for-transactions.md:88]:
```suggestion
1. Collect purchases regularly, for example every midnight, and store the data in a local database. In the request parameters, set the `startDate` to the last time of fetching purchases.
```

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+    ```
+      
+    Example:
+    
+    The following example request fetches purchase information from 19 November, 2020.
+    ```
+    /purchases/v2/?startDate=2020-11-19
+    ```
+
+2. In the local database, find the purchase by iterating through each purchase's payments list and searching for a payment with the same `uuid` as the transaction's `originatingTransactionUuid` that you saved in [Step 1: Find the transaction UUID with the Finance API](#step-1-find-the-transaction-uuid-with-the-finance-api).
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-purchase-information-for-transactions.md:101]:
```suggestion
2. In the local database, find the purchase by iterating through each purchase's payments list. Search for a payment with the same `uuid` as the transaction's `originatingTransactionUuid` that you saved in [Step 1: Find the transaction UUID with the Finance API](#step-1-find-the-transaction-uuid-with-the-finance-api).
```

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+
+   > **Tip:** In the response, you can search through all payments in all purchases using a JSONPath `$.purchases[*].payments[*].uuid`.
+
+   Example:
+   
+   In the following example response that returns purchase information from 19 November, 2020, search for `6820265b-953e-43a7-bb65-abac1ef104bf` that is the value of `originatingTransactionUuid` of the transaction. It's the same value of `uuid` of the `payments` that belongs to the purchase with `purchaseUUID1` as `2252b412-f531-405b-a073-1b3b9f2bbdd4`.
```
Comment (AndreaFilyo) [inline finance-api/user-guides/fetch-purchase-information-for-transactions.md:107]:
```suggestion
   The following example is a response that returns purchase information from 19 November, 2020. It includes a search for `6820265b-953e-43a7-bb65-abac1ef104bf` which is the value of `originatingTransactionUuid` for the transaction. This matches the `uuid` value for the `payments` that belong to the purchase with `purchaseUUID1` value `2252b412-f531-405b-a073-1b3b9f2bbdd4`.
```

**Review by AndreaFilyo — APPROVED** [review]
Nice work, I think this is a difficult area to explain.

Added some suggestions for readability, shortened some sentences etc. Please ignore if you don't want to include them.

File: finance-api/overview.md (hunk 1 above)
Comment (Astro-Choco) [reply finance-api/overview.md:7]: Will check again in the next iteration

File: finance-api/user-guides/fetch-account-transactions.md (hunk 3 above)
Comment (Astro-Choco) [reply finance-api/user-guides/fetch-account-transactions.md:174]: Will check again in the next iteration.

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md (hunk 4 above)
Comment (Astro-Choco) [reply finance-api/user-guides/fetch-purchase-information-for-transactions.md:9]: Payment types are a bit general and independent of the APIs. So will keep it as is

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md (hunk 6 above)
Comment (Astro-Choco) [reply finance-api/user-guides/fetch-purchase-information-for-transactions.md:30]: Fixed the sentence to be aligned with the note

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md (hunk 8 above)
Comment (Astro-Choco) [reply finance-api/user-guides/fetch-purchase-information-for-transactions.md:88]: Fixed

File: finance-api/user-guides/fetch-purchase-information-for-transactions.md (hunk 10 above)
Comment (Astro-Choco) [reply finance-api/user-guides/fetch-purchase-information-for-transactions.md:107]: Since searching is an emphasised action, will keep it as is.

**Review by izsusanna — APPROVED** [review]
Looks good from a purchase perspective.

**Pull request merged**